### ISSUE TITLE
Tab stops marks elements not yet tabbed to as failed keyboard navigation

### ISSUE THREAD

**Issue by katydecorah** [microsoft/accessibility-insights-web#6097]
## Describe the bug

While testing the Insider release tab stops, if you don't tab to every element on the page then tab stops will mark the remaining elements as failed, even though they are reachable by tabbing. This does not repro in production.

**To Reproduce**

1. Go to https://www.microsoft.com/en-us/ or https://www.cnn.com/
2. Open Insider release (2.35.0), FastPass, then Tab stops
3. Turn on visual helper
4. Tab only the first few elements on the page.
5. Move focus back to FastPass and Tab stops should report "Possible failures auto-detected"

## CodePen repro example

n/a

## Expected behavior

Elements that are reachable from the tab key should not appear as a failure.

## Screenshots

In the screenshot below, I used Tab stops to tab to the first few elements. When I moved focus back to FastPass it marked all remaining elements as failures:

![](https://user-images.githubusercontent.com/2180540/193314505-d6f874e9-d440-4fa3-a8f1-f4b07925644f.png)

This is what the same steps look like on production. In the screenshot below, tab stops shows that I tabbed to 6 elements but does fail the remaining elements;

![](https://user-images.githubusercontent.com/2180540/193314756-12c0e941-e740-44ea-ad7a-73009704a165.png)

## Context (please complete the following information)

- OS Name & Version: Windows 11 Enterprise 22621.521
- AI-Web Version & Environment: Insider 2.35.0
- Browser Version: Microsoft Edge Version 105.0.1343.53 (Official build) (64-bit) (I also tried Chrome and had same results)
- Target Page: https://www.microsoft.com/en-us/

## Are you willing to submit a PR?

Probably not (vacation next week)

## Did you search for similar existing issues?

Yes

## Additional context

I'm unable to attach the exported results, but the Insider Validation OneNote has exported FastPass results under "Verify FastPass against real-world target pages" section.

**Comment by jlperkins** [comment]
Confirmed issue by reproducing it; marking this as ready for team triage.

**Comment by ferBonnin** [comment]
triaged with Peter, by design as we compare tabbable elements with elements reached with Tab Stops
